tests/unit_tests/test_depletion.py

```diff
@@ -14,58 +15,59 @@ def setUp(self):
         Set up the instantiation of the Deplete class
         '''
         self.deplete = Depletion(
-            "chain_endfb71_pwr.xml", 10, 100e-6)
+            "chain_endfb71_pwr.xml",
+            10,
+            100e-6,
+            "./examples/")
+        self.materials = openmc.Materials().from_xml("./examples/materials.xml")
+        self.micro_xs = od.MicroXS.from_csv("./examples/micro_xs.csv")
 
-    def test_read_materials(self):
+    def test_update_materials(self):
         '''
-        Test when the material file is found
+        Test that the provided compositions get written to the
+        materials.xml file correctly.
         '''
-        materials = self.deplete.read_materials("examples/")
-        assert isinstance(materials, openmc.material.Materials)
-        assert materials[0].id == 5
-        assert materials[0].name == 'assembly_1'
-        assert materials[0].temperature == 900.0
+        comps = [{922350000: 0.05, 922380000: 0.95},
+                 {551370000: 0.1, 360850000: 0.8, 541350000: 0.1},
+                 {942390000: 0.10, 942410000: 0.9}]
+        material_ids, materials = self.deplete.update_materials(
+            comps, self.materials)
+        assert materials[0].nuclides == [
+            openmc.material.NuclideTuple(
+                'U235', 0.05, 'wo'), openmc.material.NuclideTuple(
+                'U238', 0.95, 'wo')]
+        assert materials[1].nuclides == [
+            openmc.material.NuclideTuple(
+                'Cs137', 0.1, 'wo'), openmc.material.NuclideTuple(
+                'Kr85', 0.80, 'wo'), openmc.material.NuclideTuple(
+                'Xe135', 0.10, 'wo')]
+        assert materials[2].nuclides == [
+            openmc.material.NuclideTuple(
+                'Pu239', 0.10, 'wo'), openmc.material.NuclideTuple(
+                'Pu241', 0.90, 'wo')]
```

Comment:
```suggestion
        assert materials[0].nuclides == [
            openmc.material.NuclideTuple('U235', 0.05, 'wo'), 
            openmc.material.NuclideTuple('U238', 0.95, 'wo')]
        assert materials[1].nuclides == [
            openmc.material.NuclideTuple('Cs137', 0.1, 'wo'), 
            openmc.material.NuclideTuple('Kr85', 0.80, 'wo'),
            openmc.material.NuclideTuple('Xe135', 0.10, 'wo')]
        assert materials[2].nuclides == [
            openmc.material.NuclideTuple('Pu239', 0.10, 'wo'), 
            openmc.material.NuclideTuple('Pu241', 0.90, 'wo')]
```
The way pep8 chose to break this up is a bit silly and harder to read than this.